readme.md

```diff
@@ -86,9 +87,9 @@ DATADIR = "pocketsphinx/test/data"
 
 # Create a decoder with certain model
 config = Decoder.default_config()
-config.set_string('-hmm', path.join(MODELDIR, 'en-us/en-us'))
```

Comment:
This change doesn't seem correct. Our models are in pocketsphinx/model/en-us/en-us for example, not in pocketsphinx/model/en-us.